common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -371,7 +372,12 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
         if (size > maxSize) {
             // Need to grow the arrays. We take care to detect integer overflow,
             // also limit array size to ArrayList.MAX_ARRAY_SIZE.
```

Comment:
Does this comment still make sense?  Where are we checking `ArrayList.MAX_ARRAY_SIZE`?